src/schema.js

```diff
@@ -184,17 +173,12 @@ class Schema {
       const out = schema[method].Out && schema[method].Out;
       const outKey = out.length === 1 ? out[0].Name : -1;
       const fnName = toCamelCase(method);
-      const delta = this.config.protocol.delta &&
-        IGNORE_DELTA_METHODS.indexOf(method) === -1 &&
-        outKey !== -1 &&
-        outKey !== SUCCESS_KEY;
 
       api[fnName] = function generatedMethod(...params) {
         return this.session.send({
           handle: this.handle,
           method,
           params,
-          delta,
           outKey,
```

Comment:
Would be awesome if we could get rid of `outKey` here too, but I think that's probably a goal of another refactoring...